src/kOS/Suffixed/AddonList.cs

```diff
@@ -0,0 +1,36 @@
+﻿using kOS.Safe.Encapsulation;
+using kOS.Safe.Encapsulation.Suffixes;
+
+
+namespace kOS.Suffixed
+{
+    public class AddonList : Structure
+    {
+        private readonly SharedObjects shared;
+        private readonly kOS.AddOns.KerbalAlarmClock.Addon kacAddon;
+        private readonly kOS.AddOns.RemoteTech.Addon rtAddon;
+        private readonly kOS.AddOns.ActionGroupsExtended.Addon agxAddon;
+
+        public AddonList(SharedObjects shared)
+        {
+            this.shared = shared;
+            kacAddon = new kOS.AddOns.KerbalAlarmClock.Addon(shared);
+            rtAddon = new kOS.AddOns.RemoteTech.Addon(shared);
+            agxAddon = new kOS.AddOns.ActionGroupsExtended.Addon(shared);
+
+            InitializeSuffixes();
+        }
+
+        private void InitializeSuffixes()
+        {
+            AddSuffix("KAC", new Suffix<Addon>(() => kacAddon));

```

Comment:
all of the addon classes are being eagerly instantiated here. ill change them to being lazy created